src/Compilers/CSharp/Test/Syntax/Parsing/CollectionLiteralParsingTests.cs

```diff
@@ -1833,690 +2253,977 @@ public void DictionarySyntaxMissingValue()
+                        N(SyntaxKind.OpenParenToken);
+                        N(SyntaxKind.TupleElement);
                         {
-                            N(SyntaxKind.IdentifierToken, "b");
+                            N(SyntaxKind.IdentifierName);
+                            {
+                                N(SyntaxKind.IdentifierToken, "A");
+                            }
+                            N(SyntaxKind.IdentifierToken, "a");
                         }
-                    }
-                    N(SyntaxKind.QueryBody);
-                    {
-                        N(SyntaxKind.SelectClause);
+                        N(SyntaxKind.CommaToken);
+                        N(SyntaxKind.TupleElement);
                         {
-                            N(SyntaxKind.SelectKeyword);
                             N(SyntaxKind.IdentifierName);
                             {
-                                N(SyntaxKind.IdentifierToken, "a");
+                                N(SyntaxKind.IdentifierToken, "B");
                             }
+                            N(SyntaxKind.IdentifierToken, "b");
                         }
+                        N(SyntaxKind.CloseParenToken);
                     }
-                }
-            }
-            M(SyntaxKind.CommaToken);
-            N(SyntaxKind.ExpressionElement);
-            {
-                N(SyntaxKind.QueryExpression);
-                {
-                    N(SyntaxKind.FromClause);
+                    N(SyntaxKind.IdentifierToken, "LocalFunc");
+                    N(SyntaxKind.ParameterList);
                     {
-                        N(SyntaxKind.FromKeyword);
-                        N(SyntaxKind.IdentifierToken, "x");
-                        N(SyntaxKind.InKeyword);
-                        N(SyntaxKind.IdentifierName);
-                        {
-                            N(SyntaxKind.IdentifierToken, "y");
-                        }
+                        N(SyntaxKind.OpenParenToken);
+                        N(SyntaxKind.CloseParenToken);
                     }
-                    N(SyntaxKind.QueryBody);
+                    N(SyntaxKind.Block);
                     {
-                        N(SyntaxKind.SelectClause);
-                        {
-                            N(SyntaxKind.SelectKeyword);
-                            N(SyntaxKind.IdentifierName);
-                            {
-                                N(SyntaxKind.IdentifierToken, "x");
-                            }
-                        }
+                        N(SyntaxKind.OpenBraceToken);
+                        N(SyntaxKind.CloseBraceToken);
                     }
                 }
             }
-            N(SyntaxKind.CloseBracketToken);
+            N(SyntaxKind.EndOfFileToken);
         }
         EOF();
     }
 
     [Fact]
-    public void ConditionalAmbiguity1()
+    public void TopLevelDotAccess_GlobalAttributeAmbiguity1()
```

Comment:
Please move tests where possible to make the diff tool can do a better job. I think this test used to be at the top of the changed section, line 164, for example.

Reply:
sure!